Libs/RaptureCore/src/test/java/rapture/dp/WorkOrderTest.java

```diff
@@ -172,7 +172,14 @@ public void testGetJobExecsAndWorkOrdersByDay() {
         while (Kernel.getSchedule().getUpcomingJobs(ctx).isEmpty()) {
             log.info("Waiting for jobs to update...");
         }
-        ScheduleManager.manageJobExecStatus();
+        while (true) {
```

Comment:
yea ok agreed.  i'm gonna look into why theres a race condition when calling ScheduleManager "run job now" type tests.  I think it has to do w/ the in-memory constructs we use for storing things when running tests.  For example, HashMaps not being guaranteed to be updated...